Lib/idlelib/autocomplete.py

```diff
@@ -27,25 +26,20 @@
 
 class AutoComplete:
 
-    menudefs = [
-        ('edit', [
-            ("Show Completions", "<<force-open-completions>>"),
-        ])
-    ]
-
-    popupwait = idleConf.GetOption("extensions", "AutoComplete",
-                                   "popupwait", type="int", default=0)
-
-    def __init__(self, editwin=None):
+    def __init__(self, editwin):
         self.editwin = editwin
-        if editwin is not None:  # not in subprocess or test
```

Comment:
Removing '=None' and the if statement caused subprocess startup to fail.
Fixed.